.travis.yml

```diff
@@ -1,94 +1,98 @@
-# This file has been generated -- see https://github.com/hvr/multi-ghc-travis
+# This Travis job script has been generated by a script via
```

Comment:
This represents a fairly large change set. `network`'s CI was steaming along nicely before. Could you please provide some motivating information to merge these changes.

Some open questions:
- Why is this better?
- Why has the script changed so drastically?
- Should we be utilizing `new build`? The plan is to replace `build` with `new-build`, so wouldn't this script remain more stable with just `build`?